throughput/spam.go

```diff
@@ -0,0 +1,428 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+//nolint:gosec
+package throughput
```

Comment:
turned off because gosec calls out math.rand for being weak

```
throughput/spam.go:80:14: G404: Use of weak random number generator (math/rand or math/rand/v2 instead of crypto/rand) (gosec)
```